### ISSUE TITLE
Network request blocking tool is a Tab/Panel (not a drawer)?

### ISSUE THREAD

**Issue by pankajparashar** [MicrosoftDocs/edge-developer#2932]
**Link to the documentation page**:
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/network-request-blocking/network-request-blocking-tool#try-the-network-request-blocking-tool

**Description of the issue**:
The linked section refers to the **Network request blocking** tool available as a _tab_ but the screenshot shows a _drawer_. I checked the latest Edge DevTools, and it indeed opens as a new tab (or panel). The Command menu still refers it as a "Drawer", I believe because the upstream Chromium still opens the tool in a drawer.

To summarise there are two issues:
- The screenshot in the article needs to be updated to show the **Network request blocking** as a tab
- The Command Menu in the Edge DevTools should show the **Network request blocking** tool as a Panel.

**Screenshots**:
<img width="840" alt="image" src="https://github.com/MicrosoftDocs/edge-developer/assets/38640616/5050abee-18e9-4495-abe9-a883288a6cbf">

**Microsoft Edge version**:
Version 119.0.2151.72 (Official build) (arm64)

**Comment by captainbrosset** [comment]
The **Network Request Blocking** tool opens at the bottom, in the drawer, for me.

Can you try reseting your settings please? Press `F1`, scroll down, click **Restore defaults and refresh**, and then try again.

You might have moved the tool to the main toolbar in the past, and this state got persisted.

**Comment by pankajparashar** [comment]
ah you're right. Apologies, I must have moved it to the Activity bar, not sure when. Didn't even know, that this was possible.

**Comment by captainbrosset** [comment]
Cool, thanks for confirming, and glad this isn't a bug.
You raise a good point though, that there's a possibility that people reading the docs get confused when they don't match their own experiences. There's so many settings that it's very possible that you have a different-looking UI than what's documented.
We document the default only, because that's most probably what the vast majority of users will see.
